### PR TITLE
[fix] : add cname sudipkundu999.github.io 

### PR DESCRIPTION
**Pull request by sudipkundu999**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://sudipkundu999.github.io/sudipkundu/

### CODE REVIEW

File: domains/sudip.json [hunk 1]
```diff
@@ -7,6 +7,6 @@
         "twitter": "sudipkundu999"
     },
     "record": {
-        "URL": "https://sudipkundu999.github.io/sudipkundu"
+        "CNAME": "sudipkundu999.github.io"
```
Comment (DEV-DIBSTER) [inline domains/sudip.json:10]: Got to have a bit more content than that.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/sudip.json (hunk 1 above)
Comment (sudipkundu999) [reply domains/sudip.json:10]: Hey, this is the site, I want to host on sudip.is-a.dev - 
https://sudipkundu999.github.io/sudipkundu/

issue was that both were redirecting to each other

Comment (sudipkundu999) [reply domains/sudip.json:10]: @DEV-DIBSTER can you please help here?

Comment (sudipkundu999) [reply domains/sudip.json:10]: Adding more context,

GitHub pages site for any given repo is
`https://<user-name>.github.io/<repo-name>`
Each of these pages are separate GitHub pages to Github even though it's under same domain.

so a user can have as many GitHub pages as they want but the domain will be the same and the path will be different.

so CNAME for all individual GitHub pages of a user will be the same - `<user-name>.github.io`

**Review by wdhdev — APPROVED** [review]

**Pull request merged**